queries/http/injections.scm

```diff
@@ -13,16 +13,12 @@
   (#set! injection.language "graphql"))
 
 ; Script (default to javascript)
-((script) @injection.content
-  (#offset! @injection.content 0 2 0 -2)
-  (#set! injection.language "javascript"))
-
-; Script with other languages
-((comment
+((#set! injection.language "javascript")
```

Comment:
can you move this downwards? i.e. after the `(_ (script))` block

Reply:
Done. I thought it won’t work.
I put it there to overwrite after setting `injection.language` as javascript, but seems like order doesn’t matter?

Reply:
I think the order doesn't matter, but now for instance I only get javascript injections. I am using
```http
# vim: ft=http
### post-request script
GET https://jsonplaceholder.typicode.com/posts/3

# @lang=lua
> {%
local body = vim.json.decode(response.body)
client.global.set("postId", body.id)
%}

### run request with variable

GET https://jsonplaceholder.typicode.com/posts/{{postId}}

> {%
  client.global.set("auth_token", response.body.jwt);
%}
```

I think `javascript` and generalized injections should be two separate queries, if you can manage to get them to always be disjoint

Reply:
Did you tested with latest http parser?
I updated the parser itself too.

![IMG_1344](https://github.com/user-attachments/assets/4795fb7a-3c74-4a3a-8634-9719c37b8916)

I opened your snippet with latest http parser installed manually but the language injection seems to be working. No error node in `:InspectTree` and `:Inspect` points to lua captures.

Reply:
Oh oops, nah didnt update the parser sorry. I'll test again in one moment